README.md

```diff
@@ -1,17 +1,23 @@
 # Electronic Prescription Service API
-
 ![Build](https://github.com/NHSDigital/electronic-prescription-service-api/workflows/Build/badge.svg?branch=master)
```

Comment:
Duplicate of the below?

Reply:
my bad resolving merge conflicts - will fix